mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -3226,18 +3228,24 @@ struct Conv1DGenerator
   }
 
   // Create a contraction: lhs{n, w, c} * rhs{c, f} -> res{n, w, f}
-  Value conv1dSliceAsContraction(RewriterBase &rewriter, Location loc,
-                                 Value lhs, Value rhs, Value res) {
+  Value
+  conv1dSliceAsContraction(RewriterBase &rewriter, Location loc, Value lhs,
+                           Value rhs, Value res,
+                           std::optional<vector::CombiningKind> maybeKind) {
```

Comment:
I think we don't need to add the `maybeKind` argument because it is already captured as a data member.